app/src/main/java/org/oppia/android/app/activity/ActivityComponent.kt

```diff
@@ -122,6 +126,14 @@ interface ActivityComponent {
   fun inject(splashActivity: SplashActivity)
   fun inject(splashTestActivity: SplashTestActivity)
   fun inject(stateFragmentTestActivity: StateFragmentTestActivity)
+  fun inject(
```

Comment:
Please inject these in alphabetical order.

Reply:
Done.